CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 :warning: **Important**: If you are upgrading to version 2.3.0 or later and have firewall rules or network configuration that blocks any unknown traffic by default, you need to update your configuration to allow connections to the new DNS names and IP addresses. Please refer to this [changelog](#230-january-23-2023) for more details.
 
+2.17.1 (in progress)
+====================
+
+Improvements
+------------
+
+- Improved websocket connection rate limiting by honoring `retryAfter` parameter during signaling connection degradation
```

Comment:
We need to provide more details here but ok for now. I'll draft a changelog tomorrow.